licenses/LICENSE-rock

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Canonical Ltd.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is it expected to be here?

Reply:
That I am not sure.  That is exactly what was included in their license.  Is that appropriate to change?